README.md

```diff
@@ -79,6 +79,13 @@ However, it's still possible to style various wavesurfer.js elements with CSS vi
 For example:
 
 ```css
+#waveform ::part(wrapper) {
```

Comment:
I’d not advertise it too much. As I mentioned in one of the PRs you linked, styling the wrapper can mess with wavesurfer’s operation, so better not to put it on the front page.